project/plugins.sbt

```diff
@@ -1,9 +1,7 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.1")
+addSbtPlugin("io.spray" % "sbt-revolver" % "0.7.2")
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.7.0")
```

Comment:
It also doubles as a reminder to myself that I need to look at the stylesheet @gauravs90 put together ;)